src/main/java/uk/gov/hmcts/darts/audit/service/AuditService.java

```diff
@@ -8,4 +9,6 @@ public interface AuditService {
 
     void recordAudit(AuditActivity activity, UserAccountEntity userAccountEntity, CourtCaseEntity courtCase);
 
+    @Transactional
```

Comment:
No need for `@Transactional` on the interface.
 The Spring team's recommendation is that you only annotate concrete classes with the @Transactional annotation, as opposed to annotating interfaces

Reply:
Agreed updated :) (I think this was just inteliJ auto adding it :D)